CIFAR/data.swift

```diff
@@ -0,0 +1,61 @@
+import Glibc
+import Python
+import TensorFlow
+
+func maybeDownload(to directory: String = ".") {
+    if (access("\(directory)/cifar-10-batches-py", F_OK) == -1) {
+        print("downloading data...")
+        system("wget -nv -O- https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz | tar xzf - -C \(directory)")
+    }
+}
+
+func loadBatches(from file: String, in directory: String = ".")
+    -> (Tensor<Int32>, Tensor<Float>) {
+    maybeDownload(to: directory)
+    let np = Python.import("numpy")
+    let pickle = Python.import("pickle")
+    let path = "\(directory)/cifar-10-batches-py/\(file)"
+    let f = Python.open(path, "rb")
+    let res = pickle.load(f, encoding: "bytes")
+
+    let bytes = res[Python.bytes("data", encoding: "utf8")]
+    let labels = res[Python.bytes("labels", encoding: "utf8")]
+
+    let labelTensor = Tensor<Int64>(numpy: np.array(labels))!
+    let images = Tensor<UInt8>(numpy: bytes)!
+    let imageCount = images.shape[0]
+
+    // reshape and transpose from the provided N(CHW) to TF default NHWC
+    let imageTensor = Tensor<Float>(images
+        .reshaped(to: [imageCount, 3, 32, 32])
+        .transposed(withPermutations: [0, 2, 3, 1]))
+
+    return (Tensor<Int32>(labelTensor), imageTensor / Float(255.0))
+}
+
+func loadTrainingBatches() -> (Tensor<Int32>, Tensor<Float>) {
+  let data = (1..<6).map { loadBatches(from: "data_batch_\($0)") }
+  return (Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.0 }),
+  Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.1 }))
```

Comment:
Indent:
```suggestion
  return (Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.0 }),
          Raw.concat(concatDim: Tensor<Int32>(0), data.map { $0.1 }))
```

---

We should probably add a wrapper around `Raw.concat` in the stdlib (taking an array of tensors-to-concatenate). Currently there's only the binary [`Tensor.concatenated(with: Tensor)`](https://github.com/apple/swift/blob/9ffbeee115e6da724a171f4aa58b318dcea16a7a/stdlib/public/TensorFlow/Ops.swift#L687) (or [`x ++ y`](https://github.com/apple/swift/blob/9ffbeee115e6da724a171f4aa58b318dcea16a7a/stdlib/public/TensorFlow/Ops.swift#L706)).